recipes/stereo-pipeline/meta.yaml

```diff
@@ -0,0 +1,182 @@
+{% set name = "stereo-pipeline" %}
+
+{% set version = "3.5.0_minimal" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NeoGeographyToolkit/StereoPipeline/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d7e11a797818812c296e55f67e19375e8d3fe35e6e6b8d3964624d4e2a1b561c
+
+build:
+  number: 0
+  skip: true  # [win]
+  
+requirements:
+  build:
+    - cmake >=3.15
+    - make
+    - c-compiler 1.8.0
+    - cxx-compiler 1.8.0
+    - {{ stdlib('c') }}
+    - wget
+
+  host:
+    - libgdal
+    - llvm-openmp
+    - flann
+    - openssl
+    - libssh2
+    - blas
+    - blas-devel
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+    - libopenblas
+    - openblas
+    - libopencv
+    - opencv
+    - proj
+    - hdf5
+    - libzlib
+    - zlib
+    - libboost-devel <1.85
+    - libjpeg-turbo
+    - libtiff
+    - libpng
+    - lz4-c
+    - libstdcxx  # [linux]
+    - libgcc  # [linux]
+    - libcxx  # [osx]
+    # The packages below will go to the stereo-pipeline feedstock
+    - chrpath  # [linux]
+    - libglvnd  # [linux]
+    - parallel
+    - pbzip2
+    - krb5
+    - pdal
+    - libpdal-core
+    - libpdal
+    - ale
+    - usgscsm
+    - eigen
+    - ceres-solver
+    - cspice =67
+    - csm >=3.0.3,<3.0.4
+    - libcurl
+    - curl
+    - gflags
+    - glog
+    - geotiff
+    # - libprotobuf >=4,<5
+    # - protobuf >=4,<5
+    - libxml2
+    - mesalib
+    - mysql
+    - nlohmann_json
+    # The pcl package fails no matter what
+    # - pcl 
+    # - python
+    # - python_abi
+    - qt-main >=5.15,<5.16
+    - qwt <6.3.0
+    - tbb
+    - tbb-devel
+    - xerces-c >=3.2.4,<3.3.0a0
+    - xorg-kbproto
+    - xorg-libice
+    - xorg-libsm
+    - xorg-libx11
+    - xorg-libxi
+
+  run:
+    - libgdal
```

Comment:
If this were trimmed down to `libgdal-core` it might help cut down the number of packages this pulls in. Same for `libpdal-core`...

Feel free to add @hobu as a maintainer here too. I can help with this once it's a feedstock.

Reply:
Noted, thank you. I left this one for dead. 

I will circle back and add you as maintainer as well once the overlords review and pull it in.

Reply:
> Unless you are part of the club and can help.

No I can't create feedstocks, but I do maintain a bunch of the geo-related ones. @ocefpaf has been extremely invaluable helping shepherd all of us to do that.